install/kubernetes/cilium/templates/hubble-ui/configmap.yaml

```diff
@@ -1,10 +1,8 @@
-{{- if and (or .Values.hubble.enabled .Values.hubble.ui.standalone.enabled) .Values.hubble.ui.enabled }}
```

Comment:
I assume this line was accidentally removed?

Reply:
Yes, nice catch, thanks Andre!